activerecord/test/cases/persistence_test.rb

```diff
@@ -912,6 +922,17 @@ def test_class_level_destroy
     assert_raise(ActiveRecord::RecordNotFound) { Reply.find(should_be_destroyed_reply.id) }
   end
 
+  def test_class_level_destroy_is_affected_by_scoping
+    should_be_destroyed_reply = Reply.create("title" => "hello", "content" => "world")
```

Comment:
Yeah, I thought the same concern. I kept the name since the same name used in `test_class_level_delete` even though a replay is not destroyed. I'll change the misleading names.